aws/resource_aws_ec2_capacity_reservation.go

```diff
@@ -98,6 +99,16 @@ func resourceAwsEc2CapacityReservation() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"outpost_arn": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.All(
+					validation.StringMatch(
+						regexp.MustCompile(`^arn:aws([a-z-]+)?:outposts:[a-z\d-]+:\d{12}:outpost/op-[a-f0-9]{17}$`),
+						"must match ^arn:aws([a-z-]+)?:outposts:[a-z\\d-]+:\\d{12}:outpost/op-[a-f0-9]{17}$",
+					),
```

Comment:
It would probably be best to just use `validateArn` here - I can't think of any other cases where we ensure that the ARN is for the correct service.

Reply:
changed to `ValidateFunc: validateArn,`